qCC/ccEntityAction.cpp

```diff
@@ -516,6 +516,59 @@ namespace ccEntityAction
 		return true;
 	}
 	
+    //! Interpolate scalar fields from on entity and transfer them to another one without the dialog
```

Comment:
from on**e**

Reply:
and don't you have a dialog still?

Reply:
You have a dialog but it is set automatically and it does not block the command line.